src/baskerville/trainer.py

```diff
@@ -459,7 +480,13 @@ def eval_step1_distr(xd, yd):
                     if first_step:
                         print("Successful first step!", flush=True)
                         first_step = False
-
+                    prog_bar.add(1)
+                    
+                    if (ei == epoch_start) and (didx < 1000) and (didx%100 == 1): 
```

Comment:
self.dataset_indexes is a list with the human and mouse datasets. so didx will either be 0 or 1.

Reply:
I think self.dataset_indexes here is the batch(step) index, as defined in the beginning of trainer.py:

        # dataset
        self.num_datasets = len(self.train_data)
        self.dataset_indexes = []
        for di in range(self.num_datasets):
            self.dataset_indexes += [di] * self.train_epoch_batches[di]
        self.dataset_indexes = np.array(self.dataset_indexes)

This script here reports a txt table inside each /f0c0/train about peak GPU memory usage at epoch 0, first 1000 steps, example below. I think this could worth keeping?:
epoch	batch	gpu_mem(GB)
0	1	15.70
0	101	15.71
0	201	15.71
0	301	15.71
0	401	15.71
0	501	15.71
0	601	15.71
0	701	15.71
0	801	15.71
0	901	15.71

Reply:
Oh yea, you're right!